pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -437,6 +446,30 @@ func GatherClusterNetwork(i *Gatherer) func() ([]record.Record, []error) {
 	}
 }
 
+// GatherHostSubnet collects HostSubnet information
+//
+// The Kubernetes api https://github.com/openshift/client-go/blob/master/network/clientset/versioned/typed/network/v1/hostsubnet.go
+// Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#hostsubnet-v1-network-openshift-io
+//
+// Location in archive: config/hostsubnet/
+func GatherHostSubnet(i *Gatherer) func() ([]record.Record, []error) {
+	return func() ([]record.Record, []error) {
+
```

Comment:
It's a real nitpick, but is the empty line here on purpose? I.O. is generally rather condensed, so it's quite striking when there's an empty line at the beginning or the end of a code block. I don't really mind it, just pointing it out.

Reply:
I guess it's not. I'll remove. Thanks.